### ISSUE TITLE
DEFAULT.QC_Accessibility_Eservices - Defect # 491 - colour contrast

### ISSUE THREAD

**Issue by marklunn** [epimorphics/ukhpi#70]
Description: 
Check that colour contrasts on the screen have sufficient contrast.

From Accessibility toolbar 'Select' Colour then Juicy Studio Contrast Analyser then Contrast Analyser All Tests

Expected: 
A new window will open with a title of Colour Contrast Analyser. 

The new window will display a tabular report.

Check the report for any rows highlighted in yellow.

There are 3 exclusion rows that may be identified in the report, check that they match the attached document and if they do the test passes. (Green text on grey image and maroon text on grey image)

#516714   #eaeaea
#990000   #cdcdcd
#990000   #d7d7d8 

Any other fails returned in the report should have a defect raised.

Actual: 
Single failure on colour contrast, report attached.

**Comment by marklunn** [comment]
![bug_491_clb_picture_150](https://cloud.githubusercontent.com/assets/13219344/17769813/69da6d42-6532-11e6-9db2-49e5abb6f540.jpeg)
[BUG_491_Colour Contrast Analyser - data page on entry.docx](https://github.com/epimorphics/ukhpi/files/424593/BUG_491_Colour.Contrast.Analyser.-.data.page.on.entry.docx)

**Comment by ijdickinson** [comment]
@marklunn I can't tell which element is failing

**Comment by marklunn** [comment]
The Orange colour needs to be tweaked slightly to meet standards.  Essentially a duplicate of defect #78

**Comment by marklunn** [comment]
Closing due to duplication with #78
